### PR TITLE
Add post-processors for the fuel column model in Serpent

### LINKED ISSUE #44: Add post-processors for the fuel column model in serpent
The fuel-column model in Serpent is missing the python functions that produce the figures with the axial flux.
This issue can be closed when those functions are added to the repo.

### PR DESCRIPTION
**Pull request by robfairh**
Fixes #44 and also adds figures of the geometries.

### CODE REVIEW

**Review by samgdotson — CHANGES_REQUESTED** [review]
This whole ``postprocessing.py`` file looks identical to the same function in ``full-core/auxiliary.py`` file.

Instead of duplicating this functionality, put the functions in a top level directory and import functions. E.g.

```python
from postprocessing import plot_serpent_axial_collapse
import os 

# Adds path to PYTHONPATH for importing.
os.path.join("~/mhtgr350-benchmark/", 0)
```

**Comment by robfairh** [comment]
Okay, I did it, but I am not sure I like the way it is right now.
So, I used:
```
sys.path.insert(1, "/home/roberto/github/mhtgr350-benchmark")
```

However, by doing that, the import of auxiliary.py comes after those lines, and pep8 doesn't like it.
So, to solve pep8 I had to create another script (pathmagic.py) for doing the import:

```
import sys

sys.path.insert(1, "/home/roberto/github/mhtgr350-benchmark")
```

and then, I do ```import pathmagic``` in my original .py file.

I did what was suggested in [this post](https://stackoverflow.com/questions/36827962/pep8-import-not-at-top-of-file-with-sys-path).

File: serpent-moltres/full-core/pathmagic.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import sys
+
+sys.path.insert(1, "/home/roberto/github/mhtgr350-benchmark")
```
Comment (samgdotson) [inline serpent-moltres/full-core/pathmagic.py:3]: Do you really need an entirely separate file for this? You already import sys in ``postprocessing.py``, so you'd really just be adding one line to that file.

**Review by samgdotson — CHANGES_REQUESTED** [review]
It's better now that the functions only live in one place. I think adding ``pathmagic.py`` could be considered "cruft." Especially since you're just adding a single line of code. Put that line in ``postprocessing.py`` and then I'll merge (unless for some reason an external file is required...). 

The fact that you have two files in the repository called ``postprocessing.py`` suggests that it may be good to give it the same treatment as ``auxiliary.py``, but since it's not getting in the way of this PR I'll create a new issue for it.

File: serpent-moltres/full-core/pathmagic.py (hunk 1 above)
Comment (robfairh) [reply serpent-moltres/full-core/pathmagic.py:3]: I don't need it, but PEP8 complains. If I did that my script would read:
```
import sys
sys.path.insert(1, "/home/roberto/github/mhtgr350-benchmark")
from auxiliary import plot_serpent_axial_collapse
```
If I do that, PEP8 throws an error: ```E402 module level import not at top of file```

Comment (samgdotson) [reply serpent-moltres/full-core/pathmagic.py:3]: Okay, this will be resolved when you address issue #56

**Review by samgdotson — APPROVED** [review]

**Pull request merged**